### PR TITLE
Add InhibRx specific cell line schema

### PR DESCRIPTION
**Pull request by MiaZganjar**
Added a specific cell line annotation schema for InhibRx. @Miha-B

### CODE REVIEW

**Review by Miha-B — APPROVED** [review]

File: resolwe_bio/descriptors/Inhibrx-cell-line.yml [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+          type: basic:string
+          required: false
+        - name: ic50_crxx
+          label: IC50 CRXX
+          type: basic:decimal
+          required: false
+        - name: amax_crxx
+          label: Amax CRXX
+          type: basic:decimal
+          required:false
```
Comment (romunov) [inline resolwe_bio/descriptors/Inhibrx-cell-line.yml:301]: Space between colon and `false`.

File: resolwe_bio/descriptors/Inhibrx-cell-line.yml [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+        - name: sensitivity_crxx
+          label: Sensitivity CRXX
+          type: basic:string
+          required: fasle
```
Comment (romunov) [inline resolwe_bio/descriptors/Inhibrx-cell-line.yml:305]: `fasle` to `false`.

File: resolwe_bio/descriptors/Inhibrx-cell-line.yml [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+          type: basic:decimal
+          required: false
+        - name: two_gene_predictions
+          label: Two gene predictions
+          type: basic:integer
+          description: |
+            Class 1 as sensitive and Class 0 as resistant.
+          required:false
+          choices:
+            - label: 0
```
Comment (romunov) [inline resolwe_bio/descriptors/Inhibrx-cell-line.yml:327]: Maybe labels could be `0 (sensitive)` or `1 (resistant)`?

File: resolwe_bio/descriptors/Inhibrx-cell-line.yml [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+              value: 0
+            - label: 1
+              value: 1
+        - name: grep_score
+          label: GREP score
+          type: basic:decimal
+          required: false
+        - name: grep_calss
```
Comment (romunov) [inline resolwe_bio/descriptors/Inhibrx-cell-line.yml:335]: `grep_calss` to `grep_class`.

**Review by romunov — CHANGES_REQUESTED** [review]
Some typos, please fix and ping me when you're done so that I can check if the schema registers okay.

**Pull request closed without merging**